cpp/src/community/detail/common_methods.cuh

```diff
@@ -0,0 +1,370 @@
+/*
+ * Copyright (c) 2020-2022, NVIDIA CORPORATION.
```

Comment:
This file contains implementation that was formerly in `louvain.cuh`, so keeping the original copyright range.